src/test/java/ust/tad/ansiblempsplugin/analysis/SerializeXMLTest.java

```diff
@@ -1,59 +1,51 @@
 package ust.tad.ansiblempsplugin.analysis;
 
-import com.fasterxml.jackson.core.JsonProcessingException;
-import com.fasterxml.jackson.dataformat.xml.XmlMapper;
-import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.test.context.SpringBootTest;
-import ust.tad.ansiblempsplugin.ansiblemodel.*;
-
-import java.io.File;
-import java.io.IOException;
-import java.util.Set;
-
-import static org.junit.jupiter.api.Assertions.assertNotNull;
 
 @SpringBootTest
 public class SerializeXMLTest {
 
-    @Value("${mps.inputModel.path}")
-    private String mpsInputPath;
+  @Value("${mps.inputModel.path}")
+  private String mpsInputPath;
 
-    @Test
-    public void serializeAnsibleToXML() throws JsonProcessingException {
-        AnsibleDeploymentModel modelToSerialize = createDummyModel();
+  // TODO FIX TESTS
```

Comment:
This should be processed before the merge!

Reply:
fixed